docs/hugo/content/introduction/annotations.md

```diff
@@ -0,0 +1,27 @@
+# Annotations understood by the operator
+
+## Annotations specified by the user
+
+Note that unless otherwise specified, allowed values are _case sensitive_ and should be provided in lower case.
+
+1. `serviceoperator.azure.com/reconcile-policy`: Specifies the reconcile policy to use. Allowed values are:
+    - `run`: The operator performs all actions as normal. This is the default if no annotation is specified.
+    - `skip`: All modification actions on the backing Azure resource are skipped. GETs are still issued to ensure that the resource
+    exists. If the resource doesn't exist, the `Ready` condition will show a `Warning` state with the details of the missing resource
+    until the resource is created. If the resource is deleted in Kubernetes, it is _not_ deleted in Azure. In REST API terminology, 
+    PUT and DELETE are skipped while GET is allowed.
+    - `skip-delete`: Modifications are pushed to the backing Azure resource, but if the resource is deleted in Kubernetes, 
+    it is _not_ deleted in Azure. In REST API terminology, PUT and GET are allowed while DELETE is skipped.
+    
+    If a value other than one of the above is specified, an error is written to the operator logs and the most restrictive 
+    option (`skip`) is chosen. To allow modifications, either correct the malformed annotation on the resource or remove the 
+    annotation entirely.
```

Comment:
I like this safe default.

Reply:
I've changed this (see the conversation [here](https://github.com/Azure/azure-service-operator/pull/2060#discussion_r794972922)). I'm thinking if we want this behavior we need to rename the annotation itself to something that more strongly conveys "this will skip no matter what you set" such as `skip-reconcile` and `skip-delete`.

I'd be OK doing that, although the fact that the two annotations overlap some and you can set them both at the same time (`skip-reconcile` is basically `skip-delete` + `skip-modifications`) could be confusing. It feels a bit like two coupled booleans to me.

As I mentioned in the other comment chain, we could also do a webhook that rejects this annotation with unknown values. I'm not sure that's really a pattern that Kubernetes users are used to but it's a much nicer failure mode than the way I had this (which allows an invalid value and then makes you debug through the operator logs to determine that your value was bad).

Reply:
Fair enough.